components/styling/MovieDetailsStyling.js

```diff
@@ -82,7 +93,23 @@ export const HeadingStyledContainer = styled.div`
   text-shadow: 2px 2px 1px rgba(0, 0, 0, 0.5);
 `;
 
-export const StyledMovieReview = styled.ul`
+export const StyledMovieCardReview = styled.div`
   ${StyledCard}
   ${StyledText}
+  padding: 6%;
+`;
+
+export const StyledMovieReview = styled.div`
+  ${StyledText}
+  text-align: center;
+  margin-bottom: 0px;
 `;
+
+export const StyledSquareIcon = styled.div`
+  margin-top: 16px;
+  font-size: 10px;
+  float: right;
+`;
+
+export const StyledSquareIconImageWidth = 12;
+export const StyledSquareIconImageHeight = 12;
```

Comment:
It's a bit strange to user upper case variable names for non-components. There are some more in this file that were here before this PR. Please fix the naming for all widths and heights.

```suggestion
export const styledSquareIconImageWidth = 12;
export const styledSquareIconImageHeight = 12;
```